developer_guide.md

```diff
@@ -0,0 +1,117 @@
+# Development guidelines for Router Sidecar
+
+* The Slice Router is a network service component that provides a virtual L3 IP routing functionality within a cluster for the Slice overlay network.
+* Each slice in a cluster has one slice router with the possibility of a redundant pair option.
+* The Slice Operator manages the lifecycle of a Slice Router by overseeing the deployment, configuration, continuous monitoring, and management of the Slice Router.
+* The Slice Router provides a full mesh network connectivity between the application pods and slice gateway pods in a cluster.
+
+## Building and Installing `router-sidecar` in a Local Kind Cluster
+For more information, see [getting started with kind clusters](https://docs.avesha.io/opensource/getting-started-with-kind-clusters).
```

Comment:
The doc link has changed to, replace it with  https://docs.avesha.io/documentation/open-source/0.2.0/getting-started-with-kind-clusters